py3-httpcore.yaml

```diff
@@ -1,7 +1,7 @@
 # Generated from https://pypi.org/project/httpcore/
 package:
   name: py3-httpcore
-  version: 0.18.0
+  version: 1.0.0
```

Comment:
this version is not set as latest... yet (maybe) should we move forward anyways?

cc @imjasonh

Reply:
should be fine from the change log  and its available in pypi as latest